tests/schemes/test_thrift.py

```diff
@@ -1079,55 +1074,6 @@ def test_headers_should_be_a_map_of_strings(headers, service):
 
 @pytest.mark.gen_test
 @pytest.mark.call
-@pytest.mark.parametrize('ClientTChannel', [TChannel, DeprecatedTChannel])
```

Comment:
I think you'll have to modify the fixtures here too. The fixtures tried all 4 combinations of thrift and thriftrw.